src/components/NavigationFloating.js

```diff
@@ -57,16 +57,16 @@ const useStyles = makeStyles((theme) => {
   };
 
   const borderHoverTheme = (props) => {
-    const borderFountainBlue2 = `1px solid ${theme.custom.colors.fountainBlue2}`;
+    const borderFountainBlueLighter = `1px solid ${theme.custom.colors.fountainBlueLighter}`;
     switch (props?.color) {
       case 'turquoise':
-        return borderFountainBlue2;
+        return borderFountainBlueLighter;
       case 'violet':
         return `1px solid ${theme.custom.colors.oldLavender}`;
       case 'white':
-        return borderFountainBlue2;
+        return borderFountainBlueLighter;
       default:
-        return borderFountainBlue2;
+        return borderFountainBlueLighter;
```

Comment:
even why have the turquoise case if the default is the same?

Reply:
For a better code readibility - you know which cases are important